### DISCUSSION TITLE
How to get text content from MULTILEADER

### DISCUSSION THREAD

**Discussion by dmkh1984** [mozman/ezdxf#824] [Q&A]
Good day.
Can't find any way to get a content which stored in MULTILEADER. 
I receive errors like:
`ezdxf.lldxf.const.DXFAttributeError: Invalid DXF attribute "m_text" for entity MULTILEADER`
and seems I can get everything with https://ezdxf.readthedocs.io/en/stable/dxfentities/mleader.html#ezdxf.entities.MultiLeader from MULTILEADER only but text.
Any way to get text from MULTILEADER?

**Comment by mozman** [comment]
The `MULTILEADER` entity is very complex and as a special method to get the `MTEXT` content: [MultiLeader.get_mtext_content()](https://ezdxf.mozman.at/docs/dxfentities/mleader.html#ezdxf.entities.MultiLeader.get_mtext_content)

This method and some others were not included in the documentation, that is fixed now.
